rest/oidc_api.go

```diff
@@ -259,3 +310,19 @@ func (h *handler) getOIDCCallbackURL(providerName string, isDefault bool) string
 	}
 	return callbackURL
 }
+
+// setStateCookie sets the state cookie.
+func setStateCookie(res http.ResponseWriter, value string, httpOnly bool) {
+	http.SetCookie(res, &http.Cookie{
+		Name: stateCookieName, Value: value, Path: "/",
```

Comment:
Is there a reason to not use the db path for this cookie (as with AddDbPathToCookie)?